backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java

```diff
@@ -1077,11 +1093,21 @@ void writeDevices() {
                     writeInput();
                     break;
                 }
+            case aarch64:
+                writeInput();
+                // TODO, is there a better way to define the keyboard?
```

Comment:
Can't we just use writeInput() (maybe slighyly adapt it?)

Reply:
the current implement is sufficient. If there is no other function available, I would not change it I this PR